roles/openshift_version/meta/main.yml

```diff
@@ -13,3 +13,5 @@ galaxy_info:
   - cloud
 dependencies:
 - role: lib_utils
+- role: container_runtime
+- role: openshift_facts
```

Comment:
Not sure this is necessary here.

Reply:
I tried without openshift_facts and it was still failing

Reply:
I'm happy to openshift_facts here if you like, that's not really going to change anything in this role, and it's essentially a 0 task import.